posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+|New
+|<<coreprofile, See previous table>>
+
+|https://jakarta.ee/specifications/authentication/3.0/[Authentication 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/appAuthentication-3.0.html[appAuthentication-3.0]
+
+|https://jakarta.ee/specifications/cdi/4.0/[Context Dependency Injection 4.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+|https://jakarta.ee/specifications/concurrency/3.0/[Concurrency 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/concurrent-3.0.html[concurrent-3.0]
+
+|https://jakarta.ee/specifications/expression-language/5.0/[Expression Language 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/expressionLanguage-5.0.html[expressionLanguage-5.0]
+
+|https://jakarta.ee/specifications/faces/4.0/[Expression Language 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/faces-4.0.html[faces-4.0]
+
+|https://jakarta.ee/specifications/security/3.0/[Security 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/appSecurity-5.0.html[appSecurity-5.0]
+
+|https://jakarta.ee/specifications/servlet/6.0/[Servlet 6.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/servlet-6.0.html[servlet-6.0]
+
+|https://jakarta.ee/specifications/tags/3.0/[Standard Tag Library 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/servlet-6.0.html[pages-3.1]
+
+|https://jakarta.ee/specifications/persistence/3.1/[Persistence 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/persistence-3.1.html[persistence-3.1]
+
+|https://jakarta.ee/specifications/pages/3.1/[Server Pages 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/pages-3.1.html[pages-3.1]
+
+|https://jakarta.ee/specifications/websocket/2.1/[WebSocket 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/websocket-2.1.html[websocket-2.1]
+
+|https://jakarta.ee/specifications/bean-validation/3.0/[Bean Validation 3.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/beanValidation-3.0.html[beanValidation-3.0]
+
+|https://jakarta.ee/specifications/debugging/2.0/[Debugging Support for Other Languages 2.0]
+|No change
+|Not applicable
+
+|https://jakarta.ee/specifications/enterprise-beans/4.0/[Enterprise Beans Lite 4.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/enterpriseBeansLite-4.0.html[enterpriseBeansLite-4.0]
+
+|https://jakarta.ee/specifications/managedbeans/2.0/[Managed Beans 2.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/managedBeans-2.0.html[managedBeans-2.0]
+
+|https://jakarta.ee/specifications/transactions/2.0/[Transactions 2.0]
+|No change
+|Not applicable (see link:{url-prefix}/docs/latest/reference/javadoc/liberty-jakartaee10-javadoc.html?package=allclasses-frame.html&class=jakarta/transaction/package-summary.html[Javadoc])
+
+|===
+
+[#platform]
+== Jakarta EE Platform 10

```

Comment:
Where does "Jakarta Common Annotations 2.1" fit in to this table?

Reply:
Although Jakarta XML Binding (mentioned earlier) is optional and not included in our `jakartaee-10.0` feature, should it still be listed in this table? Might be worth checking on the other optional technologies as well...
```
    Jakarta Enterprise Beans 3.2 and earlier entity beans and associated Jakarta Enterprise Beans QL
    Jakarta Enterprise Beans 2.x API group
    Jakarta Enterprise Web Services 2.0
    Jakarta SOAP with Attachments 3.0
    Jakarta XML Web Services 4.0
    Jakarta XML Binding 4.0
```
i.e. is this a list of the technologies that make up Jakarta EE 10 or is it a list of the features which we include in our `jakartaee-10.0` feature?

Reply:
XML Binding is included in the jakartaee-10.0 Liberty convenience feature.